### PR TITLE
GIX-1353: Add pollAccounts to nns wallet and accounts

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

In case the accounts fail to load on init. Retry to fetch them in the nns wallet and accounts.

# Changes

* Add a call to `onMount` in NnsWallet.
* Add a call to `onMount` in NnsAccounts.

# Tests

* Add for NnsWallet and NnsAccounts tests.
* Fill accounts store in Wallet and Accounts tests.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/lib/pages/NnsAccounts.spec.ts [hunk 1]
```diff
@@ -141,5 +127,12 @@ describe("NnsAccounts", () => {
         container.querySelector(".tooltip-wrapper")
       ).not.toBeInTheDocument();
     });
+
+    it("should load accounts", () => {
+      const { container } = render(NnsAccounts, { props: { goToWallet } });
+
+      const article = container.querySelector("article");
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/NnsAccounts.spec.ts:134]: What is "article"? How do you know it's an account?

File: frontend/src/tests/lib/pages/NnsWallet.spec.ts [hunk 2]
```diff
@@ -94,14 +116,6 @@ describe("NnsWallet", () => {
       expect(titleRow).not.toBeNull();
     });
 
-    it("should hide spinner when selected account is loaded", async () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/NnsWallet.spec.ts:97]: Why did you remove this?

File: frontend/src/tests/lib/pages/NnsAccounts.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsAccounts.spec.ts:134]: We don't. I didn't want to change current tests with our new best practices.

This is a legacy test. I agree it should change.

File: frontend/src/tests/lib/pages/NnsWallet.spec.ts (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsWallet.spec.ts:97]: It's now called: "should show new accounts after being loaded"

I moved it to the top and instead of just checking that the spinner is not loaded, I check that a component with information is first not rendered, and then wait for it to be rendered.

**Review by dskloetd — APPROVED** [review]
